public/assets/svg/star-empty.tsx

```
@@ -0,0 +1,21 @@
+
+const StarEmpty = ({ size = 20, color = '#DDDDDD', ...props }: SvgProps) => (
+  <svg
+    xmlns='http://www.w3.org/2000/svg'
+    width={size}
+    height={size}
+    viewBox='0 0 20 20'
+    fill="none"
+    stroke={color}
+    strokeWidth="1"
+    {...props}
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)